recipes/packaging/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "packaging" %}
```

Comment:
What can I say? See this rather long and somewhat drawn out [conversation]( https://github.com/conda-forge/staged-recipes/pull/699#issuecomment-224210547 ) that continues to the end of the thread.